### ISSUE TITLE
Team Page Design

### ISSUE THREAD

**Issue by stursberg** [hpi-swt2/sport-portal#381]
Neues Design von Teampage:

Contact @svitlanakalancha and team if buttons not visible (create new team as admin to see them)

Tasks:

- [x] Create team profile section.
- [x] Create kind of sport section
- [x] Create participant element
- [x] Create participant section
- [x] Create team-management section
- [x] Add buttons for participant mangement as soon as #338 has applied fixes. (Edit: we don't need to wait for #338 as that solves a different issue; we should be independent from that)
- [x] Check mobile optimizations
- [x] Fix failing tests (if any)

This includes fixing reported bug #396

- mobile version: participant view should be just one column
- 'Sportart' is just a string right now, not an array, just display the string
- 'Noch nie war es ...' should be replaced by 'description' of team
- actions should be grayed out when no participant is selected

Header and Footer are out of scope.
Buttons for group-actions are inactive (gray), if there is no player selected.

![teampage.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46fc919b2e9282ae8aabf301b44fff9e2c4533317346f62a20d6b02abb407e560d5a4eaa41585c4df9add829f7b02e6c981e5ab38532022d8c6853beb2901eb70bbb258a4987b6b58f4dc7ee3b21f5c6.png)

![team_mobile.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46fc919b2ebd8ea68de0b346e611b2843b492d762a10e17132d0e278ec1179050d0a49fe44585944fcfa802ef2e73e218c4e17ff933c0120996048e6edc00db513bb29814e80b7ba8949c7ee3e21e48f8c5d73.png)

**Comment by jkbe** [comment]
@zhamanakov please send background image

DONE

**Comment by jkbe** [comment]
@zhamanakov please provide mockup for mobile

DONE

**Comment by jkbe** [comment]
**Estimation: L**

**Comment by deeps96** [comment]
TODO:

- [x] Fix bug of random background bottom border (Fixed by setting background-repeat to none)

**Comment by jkbe** [comment]
I edited the initial User Story description as we are not depending on #338 to be fixed. However, #396 is incorporated into this User Story as it makes sense to fix it here.

**Comment by stursberg** [comment]
![teampage_background.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46fc919b2e9282ae8ad6bd42a756f786265329776b48e13b628ae92dbc56395b4d5a1fac425b0d10aeac8e79a3b63a708a5f02e8853c5d25c02f0dbefad341eb5ee779c11ac7a4b6984dc8e23c20e68d8c4a3916051a8ef85f27ed.png)

**Comment by deeps96** [comment]
Interesting to know:
The error, where the checkbox wasn´t display properly, was due to a missing label tag, which is strictly required for the use of Materialize.
Referring to the [doc](http://materializecss.com/forms.html#checkbox), adding `<%= label_tag 'memberCheck' + member.id.to_s do %><% end %>` after `<%= check_box_tag "members[]", member.id, false, class: 'filled-in', id: 'memberCheck' + member.id.to_s %>` fixed the problem.

**Comment by stursberg** [comment]
- [x] hover effect for Bearbeiten und Löschen (font color should turn black and background should light up like with other buttons)
- [x] Show the four blue action buttons in a 2 x 2 grid at tablet screen size

**Comment by bakoe** [comment]
@jak-ing @zhamanakov While reviewing the PR, I noticed that there is no visual indication anymore about whether a team member is a team owner or not. Before the redesign, the user could somehow guess this based on the enabled/disabled state of the “Add ownership” and “Remove ownership” buttons – at least, if the user was the team owner.

Is it intended that there is absolutely no visual indication anymore showing which team member is a team owner?

**Comment by stursberg** [comment]
![Screen Shot 2018-02-05 at 17.24.25.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f66b7191a379e80c2955415d4bbe5d4a1f06a3a3dc78e2a436628f0d0beed23f0b26933a43a9b8841fbf02b325df1e87e7ba9850d2aa6c7bb3d4d216744b470a83ed5c32aa984ec95ffe4fe64552a2113563f43b.png)

**Comment by deeps96** [comment]
## Response 1
DONE

Result:

![image](https://user-images.githubusercontent.com/10452922/35823894-56590866-0ab1-11e8-9d9b-e878f1485456.png)

Are you ok with that @zhamanakov ?

**Comment by stursberg** [comment]
1. Please make team-Description text : font-size: 1.4rem.
2. Pic proportion should not be changed (we just take center of pic and overflow-hidden the rest - like in tiles.)

![Screen Shot 2018-02-05 at 22.18.50.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f66b7191a37aed0c2a59415a4ebe5d4a1f06a3a3dc78e2a436628f0d0beed23f0b26933a43a9b8841fbf02b325df1e87e7ba9850d2aa6c7bb3d4d216744b470a83ed5c3caa984ec95ffe4de4485da2123367f43b.png)
